app/views/grids/_thumbnail.html.erb

```diff
@@ -1,3 +1,4 @@
 <div class="thumbnail-grid">
   <%= render partial: 'notes/notes', locals: { notes: nodes, unpaginated: true } %>
+  <%= render partial: 'grids/embedButton' , locals: {className: className, randomSeed: randomSeed, type: type, tagname: tagname} %>
```

Comment:
Here we could make a new local variable `thumbnails: true` and use that in a conditional in `_embedButton.html.erb` to display the correct URL.